content/desktop/release-notes.md

```diff
@@ -24,11 +24,71 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/releases.md).
 
-## 4.24.2
+## 4.25.0
+
+{{< release-date date="2023-10-26" >}}
+
+{{< desktop-install all=true version="4.25.0" build_path="/" >}}
+
+### Upgrades
+
+- [Compose v2.23.0](https://github.com/docker/compose/releases/tag/v2.23.0)
+- [Docker Scout CLI v1.0.9](https://github.com/docker/scout-cli/releases/tag/v1.0.9).
+- [Kubernetes v1.28.2](https://github.com/kubernetes/kubernetes/releases/tag/v1.28.2)
+- [cri-dockerd v0.3.4](https://github.com/Mirantis/cri-dockerd/releases/tag/v0.3.4)
+- [CNI plugins v1.3.0](https://github.com/containernetworking/plugins/releases/tag/v1.3.0)
+- [cri-tools v1.28.0](https://github.com/kubernetes-sigs/cri-tools/releases/tag/v1.28.0)
+### New
```

Comment:
fwiw these are all components for K8s, so I'd either omit them or make them sub-bullets of the K8s v1.28.2 entry

Reply:
I've added subbullets!